walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,91 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList; 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List; 
+
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Could you add a description of the parameters to the query method?
I know it came as part of the setup, but it would help to know that, for example, the events parameter is the collection of all of the events for all of the ?requested? attendees. Seems like it has all people and their events (for a particular day?).
And that the request contains the attendees and the duration of the meeting.

Also what the result of the query is. For example, is it a collection of all times when all can attend and that is (a) exactly as long as needed or (b) at least as long as needed. If (a), can there be overlapping intervals in the output?